docs/ref/pg_autoctl_show_state.rst

```diff
@@ -77,14 +77,20 @@ The ``pg_autoctl show state`` outputs includes the following columns:
 
 	Hostname and port number used to connect to the node.
 
-  - LSN
+  - TLI: LSN
 
-	Postgres Log Sequence Number, which is the current position in the WAL
-	stream. This is a hexadecimal number. See `pg_lsn`__ for more
-	information.
+	Timeline identifier and Postgres Log Sequence Number, which is the
+	current position in the WAL stream. This is a hexadecimal number. See
+	`pg_lsn`__ for more information.
```

Comment:
This is a bit confusing to read, since it talks about both the TLI and the LSN. It then only goes into what the LSN means without really making clear that that's what's happening. I would suggest something like this:

> 	Postgres Timeline Identifier and Log Sequence Number.
> 
>   The current `timeline identifier`__ is incremented each time a failover happens, or
>	when doing Point In Time Recovery. A node can only reach the secondary
>	state when it is on the same timeline as its primary node.
>
>   The Log Sequence Number is the current position in the WAL stream. This is a 
>   hexadecimal number. See `pg_lsn`__ for more information.

Reply:
Good point, edited that way now.